### ISSUE TITLE
Need to mention Visual Studio Code

### ISSUE THREAD

**Issue by jamilhaddadin** [MicrosoftDocs/azure-devops-docs#9368]
I think the document is referring to Visual Studio 2012+ and ignoring Visual Studio Code.
The document has mentioned Eclipse so for comprehensiveness I recommend added Visual Studio Code

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: a1dc9a4a-7828-f40f-4cf0-e0b1f5476ff2
* Version Independent ID: 278aed60-558e-3beb-2ed4-c2517c2cd05d
* Content: [Git and TFVC version control - Azure Repos](https://docs.microsoft.com/en-us/azure/devops/repos/tfvc/comparison-git-tfvc?view=azure-devops)
* Content Source: [docs/repos/tfvc/comparison-git-tfvc.md](https://github.com/MicrosoftDocs/azure-devops-docs/blob/master/docs/repos/tfvc/comparison-git-tfvc.md)
* Product: **devops**
* Technology: **devops-code-tfvc**
* GitHub Login: @MaximMalygin
* Microsoft Alias: **v-maksim**

**Comment by aaronhallberg** [comment]
Yes, fair enough. The whole Client section could use a bit of work, but I do not suspect this doc is important enough for a big level of effort here. Should be quick enough, however, to:
1. Add Visual Studio Code to the list _Client software_ for Git.
2. Under _Visual Studio Compatibility_, future proof it a bit by just calling out _all versions of Visual Studio since Visual Studio 2013_, or something along those lines. 
3. Maybe just delete the _User Interface_ bit, rather than try and re-write it enough to make it feel modern?
